picohttp_t/picohttp_t.c

```diff
@@ -0,0 +1,330 @@
+/*
```

Comment:
I'd do a cleanup of test applications in a later change. For now I think it's okay to go ahead with duplicated code. @huitema would that be acceptable?